### PR TITLE
B2B-2072 update RN version

### PR DESCRIPTION
**Pull request by afzalahmad-front**
## Issue ticket number and link from Jira

**Please add a link to the issue here:**
fixes: https://front-finance.atlassian.net/browse/B2B-2072

_If there isn't one just a short description of where this change came from._

## Type of change

Please tick the relevant options (with an "x").

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [x] Refactor (none of the core functionality has changed)
- [ ] Chore (adding new tests, build processes, tooling, performance)
- [ ] This change requires a documentation update

## Checklist:

Please tick the options (with an "x") of the actions which you have done. For example, it may be that an issue doesn't require new tests to be written, so you would leave option 5 unchecked:

- [x] I have performed a self-review of my code and it meets the criteria specified by the ticket/issue
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation if necessary
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have not changed existing tests (if so please add a description in the PR)
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/SDKContainer.styled.ts [hunk 1]
```diff
@@ -28,7 +28,8 @@ export const styles = StyleSheet.create({
   },
 });
 
-export const isDarkMode = useColorScheme ? useColorScheme() === 'dark' : false;
+console.log('debug >>> typeof useColorScheme', typeof useColorScheme);
```
Comment (vitalii-movchan-mesh) [inline src/components/SDKContainer.styled.ts:31]: should we keep this comment?

Comment (afzalahmad-front) [reply src/components/SDKContainer.styled.ts]: removed it.

**Review by vitalii-movchan-mesh — APPROVED** [review]
looks good to me

File: package.json [hunk 2]
```diff
@@ -44,8 +45,7 @@
   "packageManager": "yarn@1.22.17",
   "peerDependencies": {
     "react": ">=16",
-    "react-native": ">=0.60",
-    "react-native-webview": "*"
```
Comment (balint-front) [inline package.json:48]: How come we were able to remove this dependency?

**Pull request merged**